Solutions/Azure Activity/Hunting Queries/PortOpenedForAzureResource.yaml

```diff
@@ -14,33 +14,34 @@ relevantTechniques:
   - T1571
   - T1496
 query: |
-
+  let lookback = 7d;
   AzureActivity
+  | where TimeGenerated >= ago(lookback)
   | where OperationNameValue has_any ("ipfilterrules", "securityRules", "publicIPAddresses", "firewallrules") and OperationNameValue endswith "write"
   // Choosing Accepted here because it has the Rule Attributes included
   | where ActivityStatusValue == "Accepted" 
   // If there is publicIP info, include it
-  | extend publicIPAddress_ = tostring(parse_json(tostring(parse_json(tostring(parse_json(Properties).responseBody)).properties)).ipAddress) 
```

Comment:
We are removing publicIPAddress_, do we not need it?

Reply:
I did not find this column value in the sample data I used. Also, this column along with other columns extracted from JSON were being calculated but was not being used. I added usage under summarize for the other columns for which I found values.